extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-mysql/src/main/java/org/apache/guacamole/auth/mysql/MySQLAuthenticationProviderModule.java

```diff
@@ -79,6 +84,37 @@ public MySQLAuthenticationProviderModule(MySQLEnvironment environment)
         // Allow use of multiple statements within a single query
         driverProperties.setProperty("allowMultiQueries", "true");
         
+        // Set the SSL mode to use when conncting
+        MySQLSSLMode sslMode = environment.getMySQLSSLMode();
+        driverProperties.setProperty("sslMode", sslMode.toString());
+        
+        // Set legacy properties
+        if (sslMode == MySQLSSLMode.DISABLED)
+            driverProperties.setProperty("useSSL", "false");
```

Comment:
If `MySQLSSLMode.DISABLED` represents `useSSL=false`, is the value set for `sslMode` above correct in this case? Is specifying `sslMode` legal if `useSSL=false` is also specified?

Reply:
I think so - basically, the only case where `useSSL` will be set to `false` will be where `sslMode` is already set to `disabled`.  In every other case of `sslMode`, `useSSL` will be set to `true`, which, based on some feedback and documentation, is required for different (older) versions of MySQL and/or MariaDB.